search-app/src/cmr/search/services/query_execution/facets/facets_v2_results_feature.clj

```diff
@@ -146,18 +151,27 @@
   [_]
   nil)
 
+(defn- get-facet-fields-map
+  "Returns a map with the keys being the keys in facet-fields-list
+   and the values being the related facet size in the facets-size map. 
```

Comment:
I think there are too many indents here; the Clojure style guide is the same as the Common Lisp one for docstrings: indent two spaces all the way down, don't align with the first letter of the first line.

Reply:
Done. Thanks for pointing it out. I wan't sure why people don't align with the first letter of the first line, which looks prettier to me. Now I know I need to follow the style guide.